src/transaction/transaction_manager.cpp

```diff
@@ -80,7 +100,7 @@ timestamp_t TransactionManager::UpdatingCommitCriticalSection(TransactionContext
   LogCommit(txn, commit_time, callback, callback_arg);
   // flip all timestamps to be committed
   for (auto &it : txn->undo_buffer_) it.Timestamp().store(commit_time);
-
+  blocking_commit_ = false;
```

Comment:
Just curious, do we get a speed up if you pad this and the spin latch to be on different cache lines (insert `byte padding_[64]` in between and try it I guess)

Reply:
This is worth doing but not because of interference with the spin latch.  It improves the select benchmark by about 10% (which has no use of the spin latch).  My best guess is that it ensures that current running transactions is in a different cache line.  When we pull this out into a separate class/object, we should make sure that it occupies an entire cache line by itself.